src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -538,11 +577,41 @@ ucs_status_t uct_ib_mlx5_get_compact_av(uct_ib_iface_t *iface, int *compact_av);
  */
 ucs_status_t uct_ib_mlx5dv_arm_cq(uct_ib_mlx5_cq_t *cq, int solicited);
 
+/**
+ * Continue unzipping the CQEs. Should be used only when cq_unzip->left > 0.
+ *
+ * @param cq CQ that contains detected but unhandled zipped CQEs
+ *           (cq_unzip->left > 0).
+ *
+ * @return Next unzipped CQE.
+ */
+struct mlx5_cqe64 *uct_ib_mlx5_iface_cqe_unzip_continue(uct_ib_mlx5_cq_t *cq);
+
+/**
+ * Initialize the CQE unzipping related entities and unzip the first CQE.
+ *
+ * @param title_cqe The CQE that contains the title of the compression block.
+ * @param cq        CQ that contains the title.
+ * 
+ * @return First unzipped cqe.
+ */
+struct mlx5_cqe64 *
+uct_ib_mlx5_iface_cqe_unzip_start(struct mlx5_cqe64 *title_cqe,
+                                  uct_ib_mlx5_cq_t  *cq);
```

Comment:
declare (and define) "start" before the function "continue" since it's logical order of usage